server/controllers/assess/assessmentsController.ts

```diff
@@ -72,4 +75,17 @@ export default class AssessmentsController {
       })
     }
   }
+
+  adjudications(): RequestHandler {
```

Comment:
Ordinarily I'd probably suggest that this goes in a controller of its own with a `show` action, but I think in the interests of speed / less bootstrapping, I'm happy for this to stay as-is